### PR TITLE
Add support for native MySQL driver (mysqli)

### PR DESCRIPTION
**Pull request by mvorisek**
to allow support of simultaneous queries :)

### CODE REVIEW

File: src/Persistence/Sql/Expression.php [hunk 1]
```diff
@@ -493,6 +493,41 @@ public function __debugInfo(): array
...
+    /**
+     * @param array{string, array<string, mixed>} $render
+     *
+     * @return array{string, array<string, mixed>}
+     *
+     * @internal
+     */
+    protected function updateRenderBeforeExecute(array $render): array
+    {
+        [$sql, $params] = $render = $render;
```
Comment (DarkSide666) [inline src/Persistence/Sql/Expression.php]: probably bug here

Comment (mvorisek) [reply src/Persistence/Sql/Expression.php:510]: no bug, bug typo :)

Comment (mvorisek) [reply src/Persistence/Sql/Expression.php]: fixed and cheked all other code with `\$(\w+) = \$\1(?!\w|\(|->|\[| \?)` regex

File: src/Persistence/Sql/Mysql/Query.php [hunk 2]
```diff
@@ -16,6 +18,6 @@ class Query extends BaseQuery
 
     public function groupConcat($field, string $delimiter = ',')
     {
-        return $this->expr('group_concat({} separator [])', [$field, $delimiter]);
+        return $this->expr('group_concat({} separator \'' . str_replace('\'', '\'\'', $delimiter) . '\')', [$field]);
```
Comment (mvorisek) [inline src/Persistence/Sql/Mysql/Query.php:21]: https://github.com/php/php-src/issues/7903

was actually a hidden issue

**Pull request merged**